### ISSUE TITLE
ModuleNotFoundError: No module named 'flet.security'

### ISSUE THREAD

**Issue by Dario-Ciceri** [flet-dev/flet#2782]
Hello, I built a windows executable using "flet build windows" and in my code I'm using "from flet.security import decrypt" to decrypt sensitive data. 

At runtime I'm having this issue: 
`Traceback (most recent call last):
  File "<string>", line 40, in <module>
  File "<frozen runpy>", line 229, in run_module
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\name\AppData\Local\Temp\serious_python_temp2ee1bbae\main.py", line 3, in <module>
  File "C:\Users\name\AppData\Local\Temp\serious_python_temp2ee1bbae\src\google.py", line 2, in <module>
ModuleNotFoundError: No module named 'flet.security'
`

Should I add flet.security somewhere in requirements.txt or --include-packages ? 

Thank you.

**Comment by FeodorFitsner** [comment]
It's a bug. That library is available in `flet` package, but not in `flet_runtime`. Working on the fix!

**Comment by Dario-Ciceri** [comment]
> It's a bug. That library is available in `flet` package, but not in `flet_runtime`. Working on the fix!

Ok thanks, is there anything I could do to fix it?

**Comment by FeodorFitsner** [comment]
You can just literarily copy contents of https://github.com/flet-dev/flet/blob/main/sdk/python/packages/flet/src/flet/security.py into your app and use it.

**Comment by Dario-Ciceri** [comment]
> You can just literarily copy contents of https://github.com/flet-dev/flet/blob/main/sdk/python/packages/flet/src/flet/security.py into your app and use it.

Thank you!

**Comment by Dario-Ciceri** [comment]
Hello, if necessary I'll open a feature request, but while we're at it I'll ask you directly here, yesterday I was trying to make a 300x300 login page and then switch to the main page with a minimum size of 1024x720 and it works without any problems, however if I switch from the "big" page to the small login page what happens is that the page doesn't get small even if I do self. page.update(), self.update() or a combination of the two (I'm using a UserControl but nothing changes even if I take the main page directly). So I was wondering if it would be possible to run two pages in one application, each with its own independent properties or to have for views the same properties as a page to change its size, maybe there is a better way to do what I need and I am just complicating things. I liked the idea of being able to have a form with fixed 300x300 size to request login and open the main page with different size and go back to that 300x300 size in case of logout. 
I don't know if I explained myself well and if I made the idea.
Thank you.
